### PR TITLE
Adding konvoyconfig

### PR DESCRIPTION
**Pull request by jr0d**
This chart provides an addon to Konvoy for the purpose of exposing user configurations to the kubeaddons namespace.

### CODE REVIEW

File: staging/konvoyconfig/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# konoyconfig
```
Comment (shaneutt) [inline staging/konvoyconfig/README.md]:
```suggestion
# konvoyconfig
```

Comment (jr0d) [reply staging/konvoyconfig/README.md]: Nice catch :) And thinks!

File: staging/konvoyconfig/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: "Adds a ConfigMap to the kubeaddons namespace containing the values provided to this chart"
+name: konvoyconfig
+version: 0.0.2
```
Comment (hectorj2f) [inline staging/konvoyconfig/Chart.yaml]: where did we leave `0.0.1` :) ?

Comment (hectorj2f) [reply staging/konvoyconfig/Chart.yaml]: you need to add additional properties to be aligned with other charts. Look at the cert-manager-setup as an example.

File: staging/konvoyconfig/values.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Default values for konvoyconfig
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+# The cluster hostname is a FQDN which should resolve to the clusters ingress VIP
+clusterHostname: ""
```
Comment (hectorj2f) [inline staging/konvoyconfig/values.yaml]: I believe we will create more properties in the future within this configMap. Let's create them under an object `config`:
```
config:
    clusterHostname:
```

**Comment by jr0d** [comment]
I am guessing it's hanging at ths prompt in TC:
```

$ make ct.lint
docker run -t --rm -v /home/jared/git/helm/d2iq/charts:/charts -v /home/jared/git/helm/d2iq/charts/test/ct.yaml:/etc/ct/ct.yaml -w /charts quay.io/helmpack/chart-testing:v2.3.3 git fetch origin dev
The authenticity of host 'github.com (192.30.253.113)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? ^Cmake: *** [Makefile:83: ct.lint] Interrupt
```

**Comment by jr0d** [comment]
It was complaining about missing appVersion, which the docs say is optional.. and not relevant since this chart isn't managing any  downstream changes. I've added it, and CI is working now. Thanks!

**Comment by hectorj2f** [comment]
@jr0d Please sign your commits. This is mentioned in our CONTRIBUTING.md doc https://github.com/mesosphere/charts/blob/dev/CONTRIBUTING.md#merge-approval-and-release-process

**Comment by jr0d** [comment]
@hectorj2f hmm.. my commits are signed.. but with jrodirguez@d2iq.com ... looks like my member id for the org is still jrodriguez@mesosphere.io

**Pull request merged**